wagtail/admin/views/pages.py

```diff
@@ -331,9 +335,23 @@ def edit(request, page_id):
     page_class = content_type.model_class()
 
     page_perms = page.permissions_for_user(request.user)
+
     if not page_perms.can_edit():
         raise PermissionDenied
 
+    next_url = get_valid_next_url_from_request(request)
+
+    workflow_action = request.POST.get('workflow-action')
```

Comment:
Is this dead code? I can't see anywhere that a form field called `workflow-action` is being posted - the relevant form calls it `action`, and posts to the `workflow_action` view, not `edit`.